<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>711</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>711</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>712</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>380</width>
      <height>637</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>382</width>
        <height>432</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>380</width>
           <height>430</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>430</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="rider_img.qrc">:/rider_img/yesom_resize_start_dil2.png</pixmap>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>450</y>
        <width>382</width>
        <height>171</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="makeid_btn">
         <property name="minimumSize">
          <size>
           <width>380</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>380</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>13</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>지금 바로 가입하기</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="login_btn">
         <property name="minimumSize">
          <size>
           <width>380</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>380</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>13</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>로그인</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>개인정보 처리방침</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>위치기반 서비스 이용약관</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="return_btn">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: white;       /* 배경: 흰색 */
    color: #28a745;                /* 글씨: 초록 */
    border: 2px solid #28a745;     /* 테두리: 초록 */
    border-radius: 8px;            /* 둥근 테두리 */
    font-weight: bold;
}

QPushButton:hover {
    background-color: #e8f5e9;     /* hover 시 살짝 연한 초록 배경 (선택사항) */
}

QPushButton:pressed {
    background-color: #c8e6c9;     /* 클릭 시 더 진한 초록 배경 (선택사항) */
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="rider_img.qrc">
            <normaloff>:/rider_img/left.png</normaloff>:/rider_img/left.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="login_text">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>13</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>로그인</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QPushButton" name="go_login">
          <property name="minimumSize">
           <size>
            <width>360</width>
            <height>80</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>360</width>
            <height>80</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>13</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>인증 후 로그인</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="minimumSize">
           <size>
            <width>360</width>
            <height>350</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>360</width>
            <height>400</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="rider_img.qrc">:/rider_img/yesom_resize_start_dil2.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="2" column="0">
         <widget class="QLabel" name="label_8">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="rider_img.qrc">:/rider_img/makeid_3.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_7">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="rider_img.qrc">:/rider_img/makeid_messagebox.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="insert_pw">
          <property name="minimumSize">
           <size>
            <width>270</width>
            <height>60</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>270</width>
            <height>60</height>
           </size>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>230</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
    border: none;
    background: white;  /* 필요 시 배경 설정 */
    color: black;        /* 글자색 */
}</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="insert_pnum">
          <property name="minimumSize">
           <size>
            <width>270</width>
            <height>60</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>270</width>
            <height>60</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
    border: none;
    background: white;  /* 필요 시 배경 설정 */
    color: black;        /* 글자색 */
}</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QWidget" name="verticalLayoutWidget_6">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>362</width>
        <height>82</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="minimumSize">
          <size>
           <width>360</width>
           <height>70</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>360</width>
           <height>70</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="rider_img.qrc">:/rider_img/yesom_resize_upbar.png</pixmap>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>351</width>
        <height>321</height>
       </rect>
      </property>
      <widget class="QLabel" name="back_ground">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>380</width>
         <height>303</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>380</width>
         <height>303</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>380</width>
         <height>303</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>369</width>
         <height>369</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="rider_img.qrc">:/rider_img/rider_main.jpg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>428</y>
        <width>359</width>
        <height>191</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>357</width>
         <height>189</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="1" column="0">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="0">
           <widget class="QLabel" name="order_num_show">
            <property name="text">
             <string>주문 번호</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="store_name_show">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>가게 명</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="yes_btn">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: #28a745;  /* 초록색 (Bootstrap Green) /
    color: white;               / 글씨색: 흰색 /
    border: none;               / 테두리 없음 /
    border-radius: 8px;         / 모서리 둥글게 /
}
QPushButton:hover {
    background-color: #218838;  / hover 시 더 진한 초록 /
}
QPushButton:pressed {
    background-color: #1e7e34;  / 클릭 시 더 진한 초록 */
}</string>
            </property>
            <property name="text">
             <string>수락</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="no_btn">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: white;       /* 배경: 흰색 */
    color: #28a745;                /* 글씨: 초록 */
    border: 2px solid #28a745;     /* 테두리: 초록 */
    border-radius: 8px;            /* 둥근 테두리 */
    font-weight: bold;
}

QPushButton:hover {
    background-color: #e8f5e9;     /* hover 시 살짝 연한 초록 배경 (선택사항) */
}

QPushButton:pressed {
    background-color: #c8e6c9;     /* 클릭 시 더 진한 초록 배경 (선택사항) */
}
</string>
            </property>
            <property name="text">
             <string>거절</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="waiting">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>대기중 .....</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>101</y>
        <width>351</width>
        <height>321</height>
       </rect>
      </property>
      <widget class="QLabel" name="back_ground_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>380</width>
         <height>303</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>380</width>
         <height>303</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>380</width>
         <height>303</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>369</width>
         <height>369</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="rider_img.qrc">:/rider_img/rider_main.jpg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>362</width>
        <height>82</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>360</width>
           <height>70</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="rider_img.qrc">:/rider_img/yesom_resize_upbar.png</pixmap>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>420</y>
        <width>351</width>
        <height>211</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="store_name">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>가게 명</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="store_addr">
           <property name="text">
            <string>가게주소</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="order_num">
           <property name="text">
            <string>주문번호</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="request">
           <property name="text">
            <string>요청사항</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="finish_pickup">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #28a745;  /* 초록색 (Bootstrap Green) /
    color: white;               / 글씨색: 흰색 /
    border: none;               / 테두리 없음 /
    border-radius: 8px;         / 모서리 둥글게 /
}
QPushButton:hover {
    background-color: #218838;  / hover 시 더 진한 초록 /
}
QPushButton:pressed {
    background-color: #1e7e34;  / 클릭 시 더 진한 초록 */
}</string>
         </property>
         <property name="text">
          <string>픽업 완료</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QWidget" name="widget_3" native="true">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>101</y>
        <width>351</width>
        <height>291</height>
       </rect>
      </property>
      <widget class="QLabel" name="back_ground_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-20</y>
         <width>380</width>
         <height>303</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>380</width>
         <height>303</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>380</width>
         <height>303</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>369</width>
         <height>350</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="rider_img.qrc">:/rider_img/rider_main.jpg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>362</width>
        <height>82</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="minimumSize">
          <size>
           <width>360</width>
           <height>70</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>360</width>
           <height>70</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="rider_img.qrc">:/rider_img/yesom_resize_upbar.png</pixmap>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>410</y>
        <width>351</width>
        <height>221</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <widget class="QLabel" name="coust_addr">
           <property name="text">
            <string>손님 주소</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="order_num_2">
           <property name="text">
            <string>주문번호</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="request_2">
           <property name="text">
            <string>요청사항</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="finish_dil">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #28a745;  /* 초록색 (Bootstrap Green) /
    color: white;               / 글씨색: 흰색 /
    border: none;               / 테두리 없음 /
    border-radius: 8px;         / 모서리 둥글게 /
}
QPushButton:hover {
    background-color: #218838;  / hover 시 더 진한 초록 /
}
QPushButton:pressed {
    background-color: #1e7e34;  / 클릭 시 더 진한 초록 */
}</string>
         </property>
         <property name="text">
          <string>배달 완료</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_6"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="rider_img.qrc"/>
 </resources>
 <connections/>
</ui>
